values-schema.yaml

```diff
@@ -212,12 +212,21 @@ definitions:
     type: string
   env:
     additionalProperties: false
+    examples:
+      - FOO: bar
+      - bar: foo
     patternProperties:
       '[a-zA-Z_]{1,}[a-zA-Z0-9_]*':
         maxLength: 131072
         type: string
     nullable: true
     title: Environment variables
+  files:
```

Comment:
lease follow guideline https://redkubes.atlassian.net/wiki/spaces/KB/pages/4923326699/Pattern+properties
and change pattern properties to array of items

Reply:
why? who said we want to do that in core? openapi can't deal with it, but it is beneficial for CE to stay DRY and easy